lib/publiccloud/instance.pm

```diff
@@ -649,12 +649,12 @@ Mainly stored on a remote InfluxDB on a Grafana server.
 
 sub store_boottime_db() {
     my ($self, $results) = @_;
-    return unless (get_var('_PUBLIC_CLOUD_PERF_PUSH_DATA') && $results);
+    return unless (get_var('PUBLIC_CLOUD_PERF_PUSH_DATA') && $results);
 
     my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
-    my $token = get_var('_PUBLIC_CLOUD_PERF_DB_TOKEN');
+    my $token = get_var('_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN');
     unless ($url && $token) {
```

Comment:
PLease remember that this token is for **perf_2** db and _different_ from the one for **perf** db in boottime.pm.
Parameters in workerconf shall be different too.
Therefore I'd suggest to rename in `tests/publiccloud/boottime.pm` the token, i.e. `_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN_1`, and update `openqa/workerconf.sls` accordingly